macros/src/func.rs

```diff
@@ -182,3 +187,16 @@ fn create_param_parser(param: &Param) -> TokenStream {
 
     quote! { let #ident: #ty = #value; }
 }
+
+/// Creates a block responsible for building a Scope.
+fn create_scope_builder(func: &Func) -> TokenStream {
```

Comment:
i know that the function above is also similar to the one in element.rs, but this one is straight-up the same right? maybe it should be deduplicated.

Reply:
Good catch, thanks.